lib/asciidoctor/cli/options.rb

```diff
@@ -43,7 +43,8 @@ def parse!(args)
 
           EOS
 
-          opts.on('-b', '--backend BACKEND', 'set output format backend (default: html5)') do |backend|
+          opts.on('-b', '--backend BACKEND', 'set output format backend: [html5, xhtml, docbook, docbook45] (default: html5)',
+                  'Other backends such as [manpage, pdf, deckjs] can be added') do |backend|
```

Comment:
Could we write the second line as follows?

```
additional backends are supported via extensions (e.g., pdf, manpage, latex)
```

Note the lowercase at the start of the sentence. That keeps it consistent with how other options are written.